### PR TITLE
Rename mocha desktop/app tests to use *.test extension

### PR DESCRIPTION
**Pull request by eessex**
Follow up to https://github.com/artsy/force/pull/3877

- Updates file names for all mocha tests in the `src/desktop/app` directory to use `*.test.*`
- x's any broken tests

One more PR to come for the `/mobile` directory...

### CODE REVIEW

File: src/desktop/apps/artsy_primer/personalize/test/client/artists.test.coffee [hunk 1]
```diff
@@ -117,7 +117,7 @@ describe 'ArtistsView', ->
 
   describe '#render', ->
     describe 'collector_level 2', ->
-      it 'renders the template with the correct copy', ->
+      xit 'renders the template with the correct copy', ->
```
Comment (damassi) [inline src/desktop/apps/artsy_primer/personalize/test/client/artists.test.coffee]: Out of curiosity, why would these tests break after this change? Was it an issue where the file wasn't running at all before?

Comment (eessex) [reply src/desktop/apps/artsy_primer/personalize/test/client/artists.test.coffee]: Yeah all the ones that are broken were not actually being run before -- currently assembling some #'s on how many were not run to confirm.

Comment (damassi) [reply src/desktop/apps/artsy_primer/personalize/test/client/artists.test.coffee]: cool cool -- that's been a problem in the past; whole files being skipped.

**Comment by erikdstock** [comment]
Thank you for starting this! Let me know if I can do anything to help on cleanup or reenabling tests.

**Comment by eessex** [comment]
I'm having some trouble getting the `desktop/apps/auctions` tests to run correctly, we can merge https://github.com/artsy/force/pull/3882 first and I will continue to debug.

**Pull request merged**